app/views/users/show.html.erb

```diff
@@ -1 +1,32 @@
-<h1>Here the details of a user: Name, Photo, Bio, and Posts</h1>
\ No newline at end of file
+<main>
+  <div class="list">
+      <div class="user">
+        <div class="user_details">
+          <div class="user_photo">
+            <%= image_tag("mybaby.png") %>
+          </div>
+          <div class="user_name">
+            <h2><%= @user.name %></h2>
+            <h4>Number of posts: <%= @user.post_counter %></h4>
+          </div>
+        </div>
+      </div>
+
+      <div class="user_bio">
+        <h3>Bio:</h3>
+        <p><%= @user.bio %></p>
+      </div>
+
+      <% @user.most_recent_posts.each_with_index do |post, index| %>
+        <div class="user_posts">
+            <h3>Post #<%=index+1%>: <%= post.title %></h3>
+            <p><%= post.text %></p>
+            <h4><span>Comment <%= post.comments_counter %></span> <span>|</span> <span>Likes <%= post.likes_counter %></span></h4>
```

Comment:
- [ ] The Posts counter, Comments counter, and Likes counter should display the number of comments, likes, and posts. kindly make sure to implement this as for now there are no numbers on the side, if there are no likes it should be 0 otherwise it should display the number of Likes, Comments, or Posts

![Screenshot_20230404_090125](https://user-images.githubusercontent.com/25227022/229743496-e83ddff1-d58e-47b1-a926-3ae29f7f00ed.png)